jetbrains-core/src/software/aws/toolkits/jetbrains/services/s3/editor/S3TreeTable.kt

```diff
@@ -58,6 +58,15 @@ class S3TreeTable(
 
             GlobalScope.launch {
                 virtualFiles.forEach {
+                    if (it.isDirectory) {
```

Comment:
Can we share this with UploadObjectAction?

Reply:
yep, shared